admin-manual/installation/installation.rst

```diff
@@ -316,10 +316,14 @@ place to store packages for multiple operating systems. Packages for both Ubuntu
    use 'demo' as the password during the install process. The password can be
    changed after the installation is complete.
 
+   The fits package need to be installed as a prerequesite, due to a circular dependencies
+   problem that is exclusive of Ubuntu 14.04.
+
```

Comment:
Nitpick: the wording "that is exclusive of Ubuntu 14.04" sounds a bit odd to me. I think one of the following would be better:

- only happens on Ubuntu 14.04
- arises only on Ubuntu 14.04
- occurs exclusively on Ubuntu 14.04